docs/man-pages/condor_release.rst

```diff
@@ -1,82 +1,101 @@
-      
-
 *condor_release*
-=================
+================
+
+Change held jobs back to idle in HTCondor.
```

Comment:
Can we remove the clause 'in HTCondor'.  That should be assumed, given we're in the condor manual.